site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,169 @@
+- `popover-hide-delay` - applied to the popover. The invoking element *or* the popover must be **not** hovered, focused, or long-pressed for this long before the popover is hidden. The default value for this property is `infinity`.
+
+It is important to "debounce" these delays. Said another way, the delays are interpreted as *the time an element must be continuously hovered or de-hovered*. The naive Javascript implementation of this behavior is to just call `setTimeout(() => show/hidePopover(), delayValue)`. I.e. after the invoking element is hovered, wait `delayValue`, and then show the popover. But that does not properly handle the situation where the user hovers the element, and then de-hovers it, all within the delay time. In that case, no popover should be shown. Similarly, once the popover is shown and the user de-hovers the popover briefly but re-hoveres it again quickly, the popover should not close.
+
+## Popover types
+
+The new hover-triggering functionality should work for all `popover` types. There are use cases, e.g. for `popover=auto`; for example, a nested menu structure where sub-menus are shown when items in the primary menu are hovered or focused.
+
+## Example
+
+```html
+<button popovertarget=menu1>Menu</button>
+<menu popover id=menu1>
+  <li>
+    <button popovertarget=submenu1 popovertargetaction=hover>
+      Sub menu 1
+    </button>
+  </li>
+  <li>
+    <button popovertarget=submenu2 popovertargetaction=hover>
+      Sub menu 2
+    </button>
+  </li>
+</menu>
+<menu class=submenu popover id=submenu1>
+  Sub-menu #1
+</menu>
+<menu class=submenu popover id=submenu2>
+  Sub-menu #2
+</menu>
+<style>
+  button { popover-show-delay: 0.5s; }
+  [popover].submenu { popover-hide-delay: 0.5s; }
+  menu { list-style-type: none; }
+  menu button { border: none; }
+</style>
+```
+
+This example implements a popover menu (`id=menu1`) which has sub-menus (`id=submenu1` and `id=submenu2`) which can be triggered simply by hovering or focusing the main menu's buttons.
+
+# `popover=hint` Behavior
+
+When a hint is shown, it behaves differently from other popover types such as `popover=auto` and `popover=manual` in a few ways:
+
+|               | `popover=auto`            | `popover=hint` | `popover=manual` |
+| ------------- | ------------------------- | -------------- | --------------   |
+| Light dismiss |      Yes                  |     Yes        |      No          |
+| Force-hides:  | Other `auto`s and `hint`s | Other `hint`s  | Nothing          |
+| Nesting:      |     Yes                   | No             | No               |
+
+First, hints should always be light dismissable. They are transient, supplementary information, so they should not require affirmative action to close them. Clicking outside or hitting ESC will close a hint.
```

Comment:
@keithamus this help?

```suggestion
First, hints should always be light dismissable. They are transient, supplementary information, so they should not require affirmative action to close them. Clicking or tapping outside, hitting the <kbd>ESC</kbd> key, or moving focus away from the invoking element will close a hint.
```

Reply:
My only concern is the last thing about moving focus away. We explicitly didn't do that for `popover=auto` because it was jarring and hard to get right. I'd say we definitely shouldn't do that for "rich hints" for all of the same reasons.

My intention was to make the "move focus away to dismiss" is handled entirely by `popover-hide-delay`. See the new text I've added, and LMK if this allays any of your concerns here.